pkg/bootconfig/bootconfig.go

```diff
@@ -83,6 +83,8 @@ func (bc *BootConfig) Boot() error {
 				}
 			}
 		}()
+		bc.KernelArgs += " cpuidle.off=1"
+		log.Printf("bc.KernelArgs : %s", bc.KernelArgs)
 		if err := kexec.FileLoad(kernel, initramfs, bc.KernelArgs); err != nil {
```

Comment:
This is a temporary workaround for Tioga Pass with coreboot, this commit should be removed.

Reply:
Removed